server/services/weather.py

```diff
@@ -38,12 +38,18 @@ def acknowledge_recommendation(demoGuid, recommendationId):
 def trigger_simulation(demoGuid):
     """
     Trigger a simulation in the given demo
+    Creates a Snow Storm in the DC area
     """
 
     try:
         payload = dict()
         payload['demoGuid'] = demoGuid
-        response = call_openwhisk('simulate', payload)
+        event = dict()
```

Comment:
what would be the equivalent field for type and category in an event coming from the Weather Data service? It would be good to have a reference for this mapping to later inject real weather events.